tsdb/executor.go

```diff
@@ -704,10 +704,10 @@ func (e *SelectExecutor) processAggregates(results [][]interface{}, columnNames
 }
 
 func (e *SelectExecutor) aggregatePointToQueryResult(p PositionPoint, tMin time.Time, call *influxql.Call, columnNames []string) []interface{} {
-	tm := time.Unix(0, p.Time).UTC().Format(time.RFC3339Nano)
+	tm := time.Unix(0, p.Time).UTC()
 	// If we didn't explicity ask for time, and we have a group by, then use TMIN for the time returned
 	if len(e.stmt.Dimensions) > 0 && !e.stmt.HasTimeFieldSpecified() {
-		tm = tMin.UTC().Format(time.RFC3339Nano)
+		tm = tMin.UTC()
 	}
 	vals := []interface{}{tm}
```

Comment:
The problem with using `interface{}`. We are not making great use of the type system (this issue has been around for a long time).